client/sync.js

```diff
@@ -96,13 +96,13 @@ const startSync = (requester) => {
     return jsRecords
   }
 
-  ipc.on(messages.FETCH_SYNC_RECORDS, (e, categoryNames, startAt, limitResponse) => {
+  ipc.on(messages.FETCH_SYNC_RECORDS, (e, categoryNames, startAt, limitResponse, platform) => {
```

Comment:
please document the `platform` parameter in client/constants/messages.js

Reply:
on second thought, is this necessary at all? at least on laptop, the background page should be able to get the platform via `window.navigator.userAgent`

Reply:
it could be ignored on laptop. I added it to manage notifications like history or preferences records on mobile. We don't sync them for now there, but if I don't poll and delete them Amazon could keep returning me them again and again. It is just a temporary till we have those records sync on mobile as well.